meson.build

```diff
@@ -0,0 +1,470 @@
+project('xdg-desktop-portal', 'c',
+         version: '1.13.0',
+         license: 'LGPLv2+',
+         default_options: ['warning_level=2'],
+         meson_version: '>= 0.59.0')
```

Comment:
I'd like to keep xdg-desktop-portal backportable to at least Debian 11 (Meson 0.56.2), and ideally Debian 10 (Meson 0.49.0) if possible. I'm sure developers of other long-term-supported distributions like RHEL and SLES would have similar requests.

Reply:
0.56.2 was an easy one, no warnings from meson. 0.49 is going to be more work, there are several warnings that need to be worked around (some of which now deprecated upstream so we'd have to do version checks). So this mostly depends on how badly we need 0.49 support.